src/js/components/settings/integrations.js

```diff
@@ -25,17 +27,20 @@ export const IntegrationConfiguration = ({ integration, onCancel, onDelete, onSa
 
   const updateConnectionConfig = ({ target: { value = '' } }) => setConnectionConfig(value);
 
+  const { configHint, title } = EXTERNAL_PROVIDER[provider];
   return (
     <>
-      <h3 className="margin-bottom-none">{EXTERNAL_PROVIDER[provider].title}</h3>
+      <h3 className="margin-bottom-none">{title}</h3>
       <div className="flexbox space-between relative" style={{ alignItems: 'flex-end', maxWidth }}>
         <TextField
           disabled={!isEditing}
           InputLabelProps={{ shrink: !!connectionConfig }}
-          label={`${EXTERNAL_PROVIDER[provider].title} connection string`}
+          label={`${title} connection string`}
+          multiline
           onChange={updateConnectionConfig}
+          rows={isEditing ? 1 : 2}
```

Comment:
Leaving "rows" blank looks a little nicer imo, as the height dynamically changes and there are no scrollbars

Reply:
good point! should now be adjusted 👌